xcodeproj/internal/bazel_integration_files/main.swift

```diff
@@ -114,7 +116,7 @@ for arg in args {
         let matches = try NSRegularExpression(pattern: #".*?/Contents/Developer)/.*"#)
             .matches(in: sdkPath, range: fullRange)
         guard let developerDir = matches.first else {
-            print("warning: Failed to parse DEVELOPER_DIR from '-sdk'")
+            logger.log("warning: Failed to parse DEVELOPER_DIR from '-sdk'")
```

Comment:
This needs to be a print though.

This file shouldn't get bundled in the project, so that's the main issue, right?